src/ls_server.rs

```diff
@@ -479,11 +484,14 @@ impl Output for StdioOutput {
 
 pub fn run_server(analysis: Arc<AnalysisHost>, vfs: Arc<Vfs>, build_queue: Arc<BuildQueue>) {
     let logger = Arc::new(Logger::new());
+    let logger2 = logger.clone();
+    logger.log(&format!("\nStarting up at time: {}\n", time::now().ctime()));
```

Comment:
Why is it important to have the startup and shutdown times? Doesn't seem important enough to warrant an extra dep to me